### PR TITLE
S1task1 create html page

### PR DESCRIPTION
**Pull request by Freschwamm**
Basic HTML, Function to display items created and small but of CSS for display purposes

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+function pokemonCardContainer($allPokemon) {
+    $pokemonNames = "";
+    foreach ($allPokemon as $pokemon) {
+        $pokemonNames .= '<div class="card-stats">'
+            . "<p>" . $pokemon['name'] . "</p>".
```
Comment (MrLukey) [inline functions.php]: `name` is capitalised in the db. Change db or variable name to resolve

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            '<img src="' . $pokemon['image'] . '" alt="">' .
+            "<p>" .$pokemon['cost'] . "</p>".
```
Comment (MrLukey) [inline functions.php]: `cost` also needs to be standardised

File: index.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon-cards', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `pokemon-card`;');
```
Comment (MrLukey) [inline index.php:7]: table name is `pokemon-cards` in the db. Resolve one or the other.

File: pokemon-cards_2021-08-02.sql [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table pokemon-cards
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `pokemon-cards`;
+
+CREATE TABLE `pokemon-cards` (
```
Comment (MrLukey) [inline pokemon-cards_2021-08-02.sql:29]: Either change table name or variable name in PHP

**Review by MrLukey — CHANGES_REQUESTED** [review]
Just need to regenerate the db file and re-commit

File: functions.php (hunk 1 above)
Comment (Freschwamm) [reply functions.php]: (Y)

Comment (Freschwamm) [reply functions.php]: Done

File: functions.php (hunk 2 above)
Comment (Freschwamm) [reply functions.php]: Done

File: pokemon-cards_2021-08-02.sql (hunk 4 above)
Comment (Freschwamm) [reply pokemon-cards_2021-08-02.sql:29]: Fixed i think

File: index.php (hunk 3 above)
Comment (Freschwamm) [reply index.php:7]: Fixed i think

**Comment by MrLukey** [comment]
All good, just need to sort out semantic names for tables & db etc

**Review by MrLukey — APPROVED** [review]
All good

**Pull request merged**